diesel_cli/src/migrations/diff_schema.rs

```diff
@@ -514,11 +514,32 @@ where
             query_builder.push_sql(",\n");
         }
         query_builder.push_sql("\t");
+
+        let is_only_primary_key =
+            primary_keys.contains(&column.rust_name) && primary_keys.len() == 1;
+
         query_builder.push_identifier(&column.sql_name)?;
-        generate_column_type_name(query_builder, &column.ty);
-        if primary_keys.contains(&column.rust_name) && primary_keys.len() == 1 {
+
+        if cfg!(all(
+            feature = "sqlite",
+            feature = "integer_primary_key_i64_for_sqlite_3_37"
+        )) && is_only_primary_key
+            && column.ty.sql_name.eq_ignore_ascii_case("BigInt")
+        {
+            let ty = ColumnType {
```

Comment:
Can we have a small comment here explaining that this is necessary to make this equivalent to the `ROWID` column? Maybe even with a link to the sqlite documentation?